linode/manager
# feat: [M3-7028] - Add AGLB Edit Route Drawer

## Description 📝
Implements the AGLB edit route drawer.

## Preview 📷
https://github.com/linode/manager/assets/125309814/e0b40211-998f-472c-bcc1-9a37082e7c2b

## How to test 🧪

### Prerequisites
- Checkout this PR
- Turn on the MSW (bottom left dev tools)

### Reproduction steps
- Go to http://localhost:3000/loadbalancers/0/routes
- Click on the action menu `...` next to a route
- Click the `Edit` button

### Verification steps 
- Observe drawer opens with:
    - Title containing specific route label
    - Input field containing current route label 
    - Current protocol selected
- Observe when click `Edit Route` network payload should contain your new options
- Run cypress tests `yarn cy:run -s cypress/e2e/core/loadBalancers/load-balancer-routes.spec.ts`

## As an Author I have considered 🤔

*Check all that apply*
